src/uiTest/resources/project.rhino.js

```diff
@@ -507,8 +507,8 @@ function Project(underlyingProject) {
 
   this.getSyncToParentStatus = function (child) {
     const snapshot = getGraphTable().getGitMacheteRepositorySnapshot();
-    return snapshot.getManagedBranchByName(child)
-      .map(p => p.asNonRoot().getSyncToParentStatus().name())
-      .getOrElse('');
+    const managedBranch = snapshot.getManagedBranchByName(child);
+    if (managedBranch === null) return '';
```

Comment:
Out of curiosity, in the existing UI tests, do we ever fall into this case?

Reply:
Did not notice it happening locally or on the CIs.

Reply:
If not, I'd consider even removing this check. There are surely many more places in this Rhino code where we don't care about nullness. In test code, it's arguably even better to get a stack trace thrown in our face in case of unexpected null dereference, then to be silently circumvented like here.